cmd/meroxa/turbine/javascript/version.go

```diff
@@ -0,0 +1,26 @@
+package turbinejs
+
+import (
+	"context"
+	"fmt"
+
+	utils "github.com/meroxa/cli/cmd/meroxa/turbine"
+)
+
+// GetVersion calls turbine-js-cli to get the version of the app.
+func (t *turbineJsCLI) GetVersion(ctx context.Context) (string, error) {
+	cmd := utils.RunTurbineJS(ctx, "version", t.appPath)
+	stdOut, err := utils.RunCmdWithErrorDetection(ctx, cmd, t.logger)
+	fmtErr := fmt.Errorf(
+		"unable to determine the version of turbine-js-cli used for the Meroxa Application at %s",
```

Comment:
turbine-js-cli -> turbine-js

Reply:
there are two different packages now... it's not just turbine-js.  it might actually be turbine-js-framework version... i'll circle back

Reply:
@jayjayjpg do you know what's most accurate here?

Reply:
@janelletavares The package itself is called `turbine-js-cli` (see also [here](https://github.com/meroxa/turbine-js/tree/master/packages/turbine-js-cli)), but I'll leave the decision on what the best wording is up to you!

The advantage of using the more specific `turbine-js-cli` wording is that a user would have a very specific error message and if they wanted to dive into the source code to see what's happening they could do so more easily. The advantage of using the `turbine-js` wording would be that we don't expose the customer to library-internal concepts that they may never have to interact with anyways

Reply:
i don't think the separation of the packages is only internal.  we specify which packages we're manipulating in logs of the upgrade command.  which customers don't have to use at all... they can manage the package.json in their app directly, which is what most of javascript developers do, right?